Dockerfile

```diff
@@ -79,9 +79,7 @@ ARG II_INSECURE_REQUESTS=
 # DFX specific metadata for dfx deps
 ARG DFX_METADATA=
 
-RUN touch src/internet_identity/src/lib.rs
-RUN touch src/internet_identity_interface/src/lib.rs
-RUN touch src/canister_tests/src/lib.rs
+RUN touch src/*/src/lib.rs
```

Comment:
I think being explicit here would be better to avoid surprises in the future. If necessary, we can introduce other checks to ensure all libs are included; at the very least would be good to be consistent between here and the `mkdir` & `touch` above

Reply:
https://dfinity.atlassian.net/browse/GIX-2025